scripts/entrypoint.sh

```diff
@@ -1,152 +1,23 @@
-#! /usr/bin/bash
+echo "inside entrypoint file"
 
 # Files for which configuration needs to change
-SERVER_DIR=dynamodb-janusgraph-storage-backend/server/dynamodb-janusgraph-storage-backend-1.1.0/
-PROPS=${SERVER_DIR}/conf/gremlin-server/dynamodb.properties
+SERVER_DIR=janusgraph-0.4.0-hadoop2/
+PROPS=${SERVER_DIR}/conf/gremlin-server/janusgraph-cassandra.properties
 GREMLIN_CONF=${SERVER_DIR}/conf/gremlin-server/gremlin-server.yaml
-GREMLIN_HOST=0.0.0.0
-UUID=$(cat /proc/sys/kernel/random/uuid)
-STORAGE_BACKEND=com.amazon.janusgraph.diskstorage.dynamodb.DynamoDBStoreManager
-USE_TITAN_IDS=true
-TITAN_IDS=titan_ids
 
 export JAVA_OPTIONS=${JAVA_OPTIONS:- -Xms512m -Xmx1400m}
-
-export JAVA_OPTIONS="$JAVA_OPTIONS -javaagent:/opt/dynamodb/$SERVER_DIR/lib/jamm-0.3.0.jar"
-
 echo "Proceeding with JAVA_OPTIONS=$JAVA_OPTIONS"
 
-sed -i.bckp 's#host: .*#host: '$GREMLIN_HOST'#' ${GREMLIN_CONF}
-
-if grep -i '^graph.unique-instance-id=' "$PROPS" 1>/dev/null; then
-    sed -i.bckp 's#graph.unique-instance-id=.*#graph.unique-instance-id='${UUID}'#' ${PROPS}
-else
-    echo "graph.unique-instance-id=${UUID}" >> ${PROPS}
-fi
-
-echo "graph.titan-version=1.1.0-SNAPSHOT" >> ${PROPS}
-
-sed -i.bckp 's#consoleReporter: .*#consoleReporter: {enabled: false}, #' ${GREMLIN_CONF}
-sed -i.bckp 's#csvReporter: .*#csvReporter: {enabled: false}, #' ${GREMLIN_CONF}
-sed -i.bckp 's#jmxReporter: .*#jmxReporter: {enabled: false}, #' ${GREMLIN_CONF}
-sed -i.bckp 's#slf4jReporter: .*#slf4jReporter: {enabled: false}, #' ${GREMLIN_CONF}
-sed -i.bckp 's#gangliaReporter: .*#gangliaReporter: {enabled: false}, #' ${GREMLIN_CONF}
-sed -i.bckp 's#graphiteReporter: .*#graphiteReporter: {enabled: false}} #' ${GREMLIN_CONF}
```

Comment:
Probable you may need to keep this to avoid unnecessary debug logs to console.

Reply:
added it back